src/editor.js

```diff
@@ -347,6 +326,11 @@ class Editor {
     
             this.$onSelectionChange = this.onSelectionChange.bind(this);
             this.selection.on("changeSelection", this.$onSelectionChange);
+
+            this.$onStartOperation = this.onStartOperation.bind(this);
```

Comment:
This is better to do in constructor to not recreate the bound function every time

Reply:
Isn't this exactly what we want, since this listener belongs to a session and should be created and cleaned up according to the session lifecycle?